olive/model/__init__.py

```diff
@@ -509,6 +527,16 @@ def script_dir(self) -> str:
     def model_script(self) -> str:
         return self.get_local_resource("model_script")
 
+    def extend_model_attributes_from_hf_config(self) -> Dict[str, Any]:
```

Comment:
>extend_model_attributes_from_hf_config [](http://example.com/codeflow?start=8&length=38)

could be in if clause?

Reply:
I do not quit understand. More explanations?

Reply:
Updated.